icaruscode/PMT/OpReco/ICARUSFlashAssAna_module.cc

```diff
@@ -436,17 +687,27 @@ void opana::ICARUSFlashAssAna::analyze(art::Event const& e) {
 
       else {
 
-        mf::LogError("ICARUSFlashAssAna")
-           << "Invalid recob::OpFlash with label"+label.label()+"\n";
-      }
+        if( fDebug) { mf::LogError("ICARUSFlashAssAna")
+           << "Invalid recob::OpFlash with label"+label.label()+"\n"; }
```

Comment:
Use `art::inputTag::encode()` to get the full label string, which might have critical hints.
```suggestion
           << "Invalid recob::OpFlash with label '" << label.encode() << "'"; }
```